.eslintignore

```diff
@@ -0,0 +1,3 @@
+**/node_modules/*
+**/*_env/*
+**/venv/*
```

Comment:
These were necessary because we have some nested package.json and /node_modules/ directories, as well as random *.js files in some /venv/ or /marionette_env/ directories. We should possibly standardize on virtualenv naming conventions.